### PR TITLE
scrape_me removed from recipe_scrapers v15+

### PR DESCRIPTION
**Pull request by bkmgit**
## Description
Replace scrape_me with scrape_html
Closes https://github.com/GourmandRecipeManager/gourmand/issues/171

## How Has This Been Tested?

Tested on Fedora 42 with recipe-scrapers 15.2.1

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/gourmand/importers/web_importer.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def import_urls(urls: List[str]) -> Tuple[List[str], List[str]]:
             urls.remove(url)
 
     for url in urls:
-        recipe = scrape_me(url)
+        recipe = scrape_html(html,org_url=url)
```
Comment (FriedrichFroebel) [inline src/gourmand/importers/web_importer.py:44]: This does not look correct - `html` is undefined here.

```suggestion
        recipe = scrape_html(html=None, org_url=url)
```

Comment (bkmgit) [reply src/gourmand/importers/web_importer.py:44]: Providing html is optional see https://github.com/hhursev/recipe-scrapers/blob/main/recipe_scrapers/__init__.py#L892

Comment (bkmgit) [reply src/gourmand/importers/web_importer.py:44]: Perhaps it is better to download the html separately as online parameter will be removed:
https://github.com/hhursev/recipe-scrapers/blob/main/recipe_scrapers/__init__.py#L937

Comment (FriedrichFroebel) [reply src/gourmand/importers/web_importer.py]: While `html` can be a string or `None`, we still have to provide it. For the other refactoring, feel free to change it here or submit a PR later on.

Comment (bkmgit) [reply src/gourmand/importers/web_importer.py]: Can submit a PR later. Created an issue https://github.com/GourmandRecipeManager/gourmand/issues/176

**Pull request merged**